Enterprise-CMCS/macpro-mdct-mcr
# Feature Flag MLR Routes

## Summary

### Description

Currently, users can navigate to `/mlr` and see the dashboard if needed. Now, if the `mlrReport` LaunchDarkly feature flag is disabled, they'll go to the 404 page.

### Related ticket

N/A

### How to test

Turn off the `mlrReport` flag in LaunchDarkly and navigate to it, should not be rendering :)

### Important updates

N/A

### Author checklist
